skipruntime-ts/examples/utils.ts

```diff
@@ -33,7 +33,11 @@ class SkipHttpAccessV1 {
     ports: number[] = [3587],
     private creds: Protocol.Creds,
   ) {
-    this.defaultPort = ports[0];
+    let defaultPort = ports[0];
+    if (defaultPort == undefined) {
+      throw new Error("At least one port is required.");
```

Comment:
Given that there is a default value for this parameter, should we use it instead?

Also, it seems weird to have an optional parameter before a mandatory one

Reply:
Done in new version. And yeah, the optional arg just wasn't optional.